common/character_interactions/00_blackmail_interactions.txt

```diff
@@ -12,7 +12,8 @@
 	is_shown = {
```

Comment:
You should check and adapt `value = age`. There are many interesting events like getting reading glasses.
![image](https://user-images.githubusercontent.com/46384992/113731481-e0fb5680-9709-11eb-95e8-3aedba2f5ab8.png)

Reply:
Will do